src/github/utils/plugins.ts

```diff
@@ -0,0 +1,56 @@
+import { GithubPlugin, isGithubPlugin, PluginConfiguration } from "../types/plugin-configuration";
+import { EmitterWebhookEventName } from "@octokit/webhooks";
+import { GitHubContext } from "../github-context";
+import { Manifest } from "../../types/manifest";
+import { Buffer } from "node:buffer";
+
+const _manifestCache: Record<string, Manifest> = {};
```

Comment:
It limits the calls during a single run, since the manifest is retrieved from a few different places, so I did that to avoid a network call each time. Since we have a 50 request limit per worker, we can hit it quite easily.

Reply:
> [paid plan](https://www.cloudflare.com/en-gb/plans/developer-platform/) is 5$ @0x4007

More motivation to close this series A...